routers/admin/users.go

```diff
@@ -205,6 +205,14 @@ func EditUserPost(ctx *context.Context, form auth.AdminEditUserForm) {
 		u.EncodePasswd()
 	}
 
+	if u.IsLocal() && len(form.UserName) > 0 && u.Name != form.UserName {
+		if err := models.ChangeUserName(u, form.UserName); err != nil {
```

Comment:
I think `models.ChangeUserName` and `models.UpdateUser` should be with a transaction.

Reply:
@Bwko any update for this?